docs/resources/logs_metric.md

```diff
@@ -59,6 +59,7 @@ Required:
 
 Optional:
 
+- `include_percentiles` (Boolean) Toggle to include/exclude percentiles for a distribution metric. Defaults to false. Can only be applied to metrics that have a `aggregation_type` of distribution.
```

Comment:
```suggestion
- `include_percentiles` (Boolean) Toggle to include/exclude percentiles for a distribution metric. Defaults to false. Can only be applied to metrics that have an `aggregation_type` of distribution.
```

Reply:
Good catch! Updated here 👌 (https://github.com/DataDog/terraform-provider-datadog/pull/1645/commits/88396003d96590c71f9ea39c718a71015f8c7d80)

Reply:
... And here, where it should have been updated in the first place 👌 (https://github.com/DataDog/terraform-provider-datadog/pull/1645/commits/117c204b19f5dfdcea6b81917fddb41dc8b7c937)